test/model/lstnet/test_lstnet.py

```diff
@@ -106,4 +106,4 @@ def test_lstnet(
     agg_metrics, item_metrics = evaluator(
         iter(tss), iter(forecasts), num_series=len(dataset.test)
     )
-    assert agg_metrics["ND"] < 1.5
+    assert agg_metrics["ND"] < 0.25
```

Comment:
Better to keep these thresholds loose, since otherwise tests will frequently fail (they run very short training)

```suggestion
    assert agg_metrics["ND"] < 0.5
```